backend/LexBoxApi/Services/HgService.cs

```diff
@@ -36,6 +36,19 @@ public async Task DeleteRepo(string code)
         await Task.Run(() => Directory.Delete(Path.Combine(_options.Value.RepoPath, code), true));
     }
 
+    public async Task SoftDeleteRepo(string code, string deletedAt)
```

Comment:
I'm considering making this change. Then for "reset" projects, the suffix would also include `__reset`.
```suggestion
    public async Task SoftDeleteRepo(string code, string deletedRepoSuffix)
```

Reply:
I like that actually, because we probably want to cleanup reset projects and deleted projects on a different schedule, so that makes sense.